includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -84,6 +85,48 @@ public function render_provider_fields() {
 			]
 		);
 
+		switch ( $this->feature_instance::ID ) {
+			case TitleGeneration::ID:
+				add_settings_field(
+					'number_of_titles',
```

Comment:
For other Provider specific settings, we've been adding the Provider ID to the ID of the field. This way we can be sure this field ID doesn't conflict with any other ID (for instance, if two Providers want to use the same setting but another Provider doesn't).
```suggestion
					static::ID . '_number_of_titles',
```